implementation/contracts/deposit/Deposit.sol

```diff
@@ -275,7 +275,7 @@ contract Deposit is DepositFactoryAuthority {
     }
 
     /// @notice                 Anyone can provide a signature that was not requested to prove fraud during funding.
-    /// @dev                    ECDSA is NOT SECURE unless you verify the digest.
```

Comment:
I kind of like this blaring reminder everywhere, is there another spot to drop it?

Reply:
As-is it's not particularly actionable anyway (what does “verify” mean?); moreover, *I believe* keep-ecdsa does the verification in question, so to both consumers and developers this isn't a relevant concern when interacting with tBTC.

Worth a double-check, of course, but this is why we removed some other instances of the warning.

Reply:
yeah this was just cleaning up what was left from the original PR [comment](https://github.com/keep-network/tbtc/pull/459#discussion_r380381916)
Warning in ECDSAKeep should be good